### PR TITLE
Update with latest plugin info.

### PR DESCRIPTION
**Pull request by xellos00**
Update with latest plugins info and fixed typos

### CODE REVIEW

**Review by meetrick — APPROVED** [review]
LGTM

**Comment by xellos00** [comment]
@qkrybin 
Can you review this document for community-plugin? as I mentioned in 
- https://github.com/dsrvlabs/vatz/pull/600#discussion_r1794288375

File: README.md [hunk 1]
```diff
@@ -1,27 +1,41 @@
...
+|   2    | vatz-plugin-avail    | https://github.com/dsrvlabs/vatz-plugin-avail    | Monitors Avail-network nodes with provided metrics                  | https://pkg.go.dev/github.com/dsrvlabs/vatz-plugin-avail    |                 |
+|   3    | vatz-plugin-near     | https://github.com/dsrvlabs/vatz-plugin-near     | Monitors NEAR-network nodes with provided metrics                   | https://pkg.go.dev/github.com/dsrvlabs/vatz-plugin-near     |                 |
+|   4    | vatz-plugin-solana   | https://github.com/dsrvlabs/vatz-plugin-solana   | Monitors Solana-network nodes with provided metrics                 | https://pkg.go.dev/github.com/dsrvlabs/vatz-plugin-solana   |                 |
+|   5    | vatz-plugin-watchers | https://github.com/dsrvlabs/vatz-plugin-watchers | Monitors the signing status of validators in Cosmos-based protocols | https://pkg.go.dev/github.com/dsrvlabs/vatz-plugin-watchers |                 |
+|   7    | vatz-plugin-flow     | https://github.com/dsrvlabs/vatz-plugin-flow     |  Monitors Flow nodes with provided metrics                          | https://pkg.go.dev/github.com/dsrvlabs/vatz-plugin-flow     |
```
Comment (qkrybin) [inline README.md:37]: there's no 6

**Review by qkrybin — APPROVED** [review]
LGTM

**Pull request merged**